runner.py

```diff
@@ -178,17 +175,25 @@ def import_metric_providers(self):
         config = GlobalConfig().config
 
         print(TerminalColors.HEADER, '\nImporting metric providers', TerminalColors.ENDC)
+
+        # if there's a more elegant way to do this, I don't know it
+        if 'common' in config['measurement']['metric-providers']:
```

Comment:
Why don't you use the get_metric_providers function here?

As I see it has to be reduced a bit and the splitting to be done manually afterwards, but then you could reuse that part of the code

Reply:
good call, did the refactor and pushed